backend/tests/v5/unit/models/loginRecords.test.js

```diff
@@ -93,6 +94,21 @@ const testSaveLoginRecord = () => {
 	});
 };
 
-describe('models/loginRecord', () => {
+const testRemoveAllUserRecords = () => {
+	describe('Remove all user login records', () => {
+		test('Should just drop the user collection within loginRecords', async () => {
+			const fn = jest.spyOn(db, 'dropCollection').mockResolvedValue(undefined);
+
+			const username = generateRandomString();
+			await expect(LoginRecord.removeAllUserRecords(username)).resolves.toBeUndefined();
+
+			expect(fn).toHaveBeenCalledTimes(1);
+			expect(fn).toHaveBeenCalledWith('loginRecords', username);
```

Comment:
we could store 'loginRecords' in a constant somewhere

Reply:
bit here or there. We already have it as constant in the code, it's just not exposed, and i don't think we want to expose it for the test

We can do that if we use it in more than one place in the tests in the future.